styleguide/components/oisc-warnings/_default.html.haml

```diff
@@ -0,0 +1 @@
+= render partial: "@citizensadvice/design-system/haml/oisc_warning", locals: { title: "This is an OISC level 2 topic", content: "Generalist advisers can give clients the information on this page, but must refer them to an immigration specialist if they need advice. <a href='#'>Check what to do if your client needs level 2 advice.</a>" }
```

Comment:
Using tiny Haml partials for the stories themselves allows us to bypass the need for `locals.rb` and write examples exactly as we would in Rails.